app/src/main/res/layout/bottom_sheet_schedule_send.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Infomaniak Mail - Android
+  ~ Copyright (C) 2024 Infomaniak Network SA
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+<com.infomaniak.mail.views.BottomSheetScaffoldingView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.bottomSheetDialogs.ScheduleSendBottomSheetDialog">
+
+    <TextView
+        android:id="@+id/title"
+        style="@style/BodyMedium"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/marginStandardMedium"
+        android:layout_marginBottom="@dimen/marginStandardMedium"
+        android:gravity="center_horizontal"
+        android:text="@string/scheduleSendingTitle" />
+
+    <com.infomaniak.mail.ui.main.thread.actions.ActionItemView
+        android:id="@+id/lastScheduleItem"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:visibility="gone"
+        app:icon="@drawable/ic_last_schedule_selected"
+        app:title="@string/lastSelectedSchedule"
+        app:visibleDivider="false"
+        tools:visibility="visible" />
+
+    <LinearLayout
+        android:id="@+id/scheduleItems"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical" />
+
+    <com.infomaniak.mail.ui.main.thread.actions.ActionItemView
+        android:id="@+id/customScheduleItem"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:icon="@drawable/ic_pen"
+        app:iconTint="?attr/colorPrimary"
+        app:showActionIcon="true"
+        app:title="@string/buttonCustomSchedule" />
```

Comment:
Make it so ActionItemView has the same behavior as it had on master regarding the icon's tint when no `app:iconTint` is provided